common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -566,50 +566,96 @@
                                 // [5, 7) or (5, 7), or (1.2345 * (2+3), 7*4 ]  - range tolerance case
                                 // = (5*2)*3 should not be used as range tolerance
-                                string = '<numericalresponse answer="' + value + '">\n';
-                                string += '  <formulaequationinput />\n';
-                                string += hintLine;
-                                string += '</numericalresponse>\n\n';
-                                return string;
+                                numericalResponseString = '<numericalresponse answer="' + firstAnswer + '">\n';
+                            } else {
+                                answerData = getAnswerData(firstAnswer);
+                                numericalResponseString = '<numericalresponse answer="' + answerData.answer + '">\n';
+                                if (answerData.default) {
+                                    numericalResponseString += '  <responseparam type="tolerance" default="' +
+                                        answerData.default + '" />\n';
+                                }
                             }
 
-                            if (isNaN(parseFloat(value))) {
-                                return false;
+                            // Additional answer case or= [e.g. or= 10]
+                            // Since answerValues[0] is firstAnswer, so we will not include this in additional answers.
+                            additionalAnswerString = '';
+                            for (i = 1; i < answerValues.length; i++) {
+                                additionalHintLine = '';
+                                additionalTextHint = extractHint(answerValues[i]);
+                                orMatch = /^or\=\s*(.*)/.exec(additionalTextHint.nothint);
+                                if (orMatch) {
+                                    additionalAnswerData = getAnswerData(orMatch[1]);
+                                    additionalAnswer = additionalAnswerData.answer;
+                                    if (additionalTextHint.hint) {
+                                        additionalHintLine = '<correcthint' +
+                                            additionalTextHint.labelassign + '>' +
+                                            additionalTextHint.hint + '</correcthint>';
+                                    }
+                                    // Do not add additional_answer if additional answer is not numerical or
+                                    // contains range tolerance case.
+                                    if (isNaN(parseFloat(additionalAnswer)) || isRangeToleranceCase(additionalAnswer)) {
+                                        continue;
+                                    }
+                                    // Additional_answer does not support responseparam (default tolerance etc.)
```

Comment:
I read in the ticket that the reason that default tolerance (and range) is not supported is that it would be more difficult. Would it really be more difficult to support default tolerance? It seems like the helper methods already exist.